### PR TITLE
Add FAQs

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# FAQs #
+
+##Cloning questions##
```
Comment (sthirugn) [inline docs/faqs.md]: The word `questions` is redundant as we are already in FAQ page.  Instead, the name should match the role name for ease of use.

Maybe you can just name this as `satellite-clone`

File: docs/faqs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+2. Use the Satellite you are cloning as the Control node. You can clone the satellite-clone repo on the existing Satellite server and use that as the Control Node. The Destination Node is a blank machine.
+
+3. Use a separate machine as the Control Node. This is a machine that is seperate from the blank machine you are cloning to and the Satellite you are cloning. The backups will have to be copied onto the server. The Destination Node is a blank machine.
+
+####Can I clone Capsules?####
+
+Capsules are not able to be cloned at this time (Feb 2017). This is a planned feature.
+
+##Hostname change questions##
```
Comment (sthirugn) [inline docs/faqs.md]: Title can be changed to `satellite-hostname`

File: docs/faqs.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Use the blank destination node as the Control node. This is the "one machine" approach. You can clone the satellite-clone repo on this server, move the backups to the specified folder on the server, and then run the playbook from the repo.
+
+2. Use the Satellite you are cloning as the Control node. You can clone the satellite-clone repo on the existing Satellite server and use that as the Control Node. The Destination Node is a blank machine.
+
+3. Use a separate machine as the Control Node. This is a machine that is seperate from the blank machine you are cloning to and the Satellite you are cloning. The backups will have to be copied onto the server. The Destination Node is a blank machine.
+
+####Can I clone Capsules?####
+
+Capsules are not able to be cloned at this time (Feb 2017). This is a planned feature.
```
Comment (sthirugn) [inline docs/faqs.md]: What do you really mean when you say `capsule cloning`?  
- I think Capsule just syncs the content and does not store anything else isnt it?.  If so, is it not better to just resync the Capsule using the new cloned Satellite?

File: docs/faqs.md (hunk 1 above)
Comment (johnpmitsch) [inline docs/faqs.md]: The only concern with putting satellite-clone here is that is the name of the repo  so it may get confusing. We could do `satellite clone playbook` I suppose, but my thought around this FAQ is to put everything in layman's terms

File: docs/faqs.md (hunk 3 above)
Comment (johnpmitsch) [inline docs/faqs.md]: We are getting alot of questions about cloning a capsule, this is taking an existing capsule, and creating an duplicate one on another machine. Capsules do sync and store content on them to make the content available to clients.

File: docs/faqs.md (hunk 2 above)
Comment (johnpmitsch) [inline docs/faqs.md]: I still would like to keep this in ordinary, everyday english. I think what I could do is add the playbook names to `Cloning questions` and here, but still keep the titles. This may sort out any confusion

**Comment by johnpmitsch** [comment]
@sthirugn updated
